packages/sdks/typescript/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@latitude-data/sdk",
-  "version": "1.1.1",
+  "version": "2.0.0",
```

Comment:
Although no one is using yet tool calling from our sdk this change is a breaking change that deserves a major version bump. Also I think is nice to introduce tool calling with a new version